src/ivc/public_params.rs

```diff
@@ -245,12 +243,14 @@ where
     RP1: ROPair<C1::Scalar, Config = MainGateConfig<MAIN_GATE_T>>,
     RP2: ROPair<C2::Scalar, Config = MainGateConfig<MAIN_GATE_T>>,
 {
+    #[instrument(name = "creating pp", skip_all)]
```

Comment:
based on your doc, we are using both "instrument" and "span" method in this PR?

Reply:
`instrument` is just a syntax sugar to create span guard inside fn